test/test.ml

```diff
@@ -105,6 +105,7 @@ let () =
     | Ok config ->
     match Context.refresh_secrets ctx with
     | Ok ctx ->
+      let%lwt () = Action_local.refresh_username_to_slack_id_tbl ~ctx in
```

Comment:
I think the test harness should be modified to refresh the context with a handcrafted mapping (in OCaml, not from file) for some github/slack profiles in the existing test cases

I would prefer to update the output of the existing test cases with the changes from this feature, not to create new ones

Reply:
I think it's probably a good idea to have a separate file for Slack profiles, since we'd probably want to test with an actual Slack `users.list` payload. I don't think the existing test cases have any message body with GitHub user mentions either, so we should keep some of the new test files.